### ISSUE TITLE
ovlmenu crashes AMS on start (10.0.4/AMS 0.13)

### ISSUE THREAD

**Issue by JanJastrow** [WerWolv/Tesla-Menu#18]
Hi there,
first of all, thanks for your cool homebrew softwares - really helpful.

I have recently updated my Switch to 10.0.4/AMS 0.13.0 and now I have problems with Tesla.
I have downloaded both latest versions of nx-ovlloader & ovlmenu.ovl from GitHub and put them in their folders on my SD card. Sadly, as soon as I start my switch, AMS crashes (shortly before the main/login screen): `Error Code 2168-0001 (0x2a8)`

According to the error screen, it's the fault of program 4200{0}07E51A which I identifed as nx-ovlloader.
After renaming ovlmenu.ovl to ovlmenu._ovl (to not autoload it), my switch boots up fine.

I could not find any config files/settings that might be related to this.

Any idea what might be the cause?
![IMG_6247](https://user-images.githubusercontent.com/2591991/85211160-9b21fd00-b346-11ea-9d82-272a90b8eca4.JPG)

**Comment by masagrator** [comment]
Is this emummc or sysnand?

**Comment by JanJastrow** [comment]
Hi, sorry that I forgot to mention it:
Yes, I'm running emummc.

_Both sysnand and emummc are on 10.0.4 (if that matters)_

**Comment by JanJastrow** [comment]
Hi again,
I have found a solution that works in my case.

While navigating my SD card through FTP I noticed that I could not open the `.overlays` folder through my FTP client. It would download "something" instead of opening the folder.
I have now recreated the folder through FTP and added 4 overlays - everything seems to work fine now. 😎

I guess the file transfer from macOS to my SD card is "different" (non-standard).
